modernization-definition.md

```diff
@@ -0,0 +1,21 @@
+# Defining Mainframe Modernization
+
+Businesses are digitally transforming to better serve their customers,
+reduce time to market, and increase their ability to respond quickly to
+market changes, consumer behavior, and new regulations. As businesses
+transform, they are imposing significant demands on existing mainframe
+applications and data, driving the need for modernization to better
+achieve business outcomes.
+
+It's often not a question of if you need to modernize mainframe
+applications, only a question of how. The OMP Working Group embraces a
+hybrid cloud approach to mainframe modernization with the goal of
```

Comment:
full range of approaches to mainframe modernization, including modernizing in place, hybrid cloud or moving to another platform.  The methods chosen need to focus on the goals of